### PR TITLE
Removed nil returns from ScanUnspendOutputs implementations.

### LINKED ISSUE #335: Remove nil returns of ScanUnspendOutputs implementation
**Describe the bug**
When i use the method ScanUnspendOutputs of hte SkycoinAddress type when i am offline this return a "Invalid memory address or nil pointer dereference" error.

- OS:  Ubuntu
- Platform: Linux 4.17.0-041700rc4-generic x86_64

**Steps to Reproduce**

1.Run the application in offline mode.
2. Go to OutputsPage.
3. See error

**Possible implementation**
Remove the return nil sentences.

### PR DESCRIPTION
**Pull request by hsequeda**
Fixes #335

**Changes:**
- Removed nil returns from ScanUnspendOutputs implementation.

**Does this change need to mentioned in CHANGELOG.md?**
no

**Requires testing**
Requires testing
no

### CODE REVIEW

File: src/models/modelWallets.go [hunk 1]
```diff
@@ -122,8 +122,8 @@ func (m *ModelWallets) loadModel() {
 
 		for addresses.Next() {
 			a := addresses.Value()
-			outputs := a.GetCryptoAccount().ScanUnspentOutputs()
-			if outputs == nil {
+			outputs, err := a.GetCryptoAccount().ScanUnspentOutputs()
+			if err != nil {
```
Comment (olemis) [inline src/models/modelWallets.go:126]: log some message at this point

**Review by olemis — CHANGES_REQUESTED** [review]

**Review by olemis — CHANGES_REQUESTED** [review]
@stdevHsequeda why u adding so many files for such a tiny task ?

**Comment by olemis** [comment]
> @stdevHsequeda why u adding so many files for such a tiny task ?

in vendor I mean

**Comment by hsequeda** [comment]
> > @stdevHsequeda why u adding so many files for such a tiny task ?
> 
> in vendor I mean

I only add the vendor

**Comment by hsequeda** [comment]
> > > @stdevHsequeda why u adding so many files for such a tiny task ?
> > 
> > 
> > in vendor I mean
> 
> I only add the vendor
Do you want me to reverse those changes?

**Comment by olemis** [comment]
@stdevHsequeda no , I wander why these added at this point ?

**Comment by hsequeda** [comment]
Review

File: src/models/modelWallets.go (hunk 1 above)
Comment (stdevMac) [reply src/models/modelWallets.go:126]: @stdevHsequeda pls solve this asap to merge this

**Review by stdevMac — APPROVED** [review]

**Pull request merged**